pkg/receiver/jobreceiver/README.md

```diff
@@ -12,20 +12,20 @@ of downloading runtime assets necessary to run a particular monitoring job.
 
 ## Configuration
 
-| Configuration | Default | Description
-| ------------ | ------------ | ------------
-| exec | required | A `exec` configuration block. See details [below](#execution-configuration)
-| schedule | required | A `schedule` configuration block. See details [below](#schedule-configuration)
-| output | | An `output` configuration block. See details [below](#output-configuration)
+| Configuration | Default | Description|
```

Comment:
I don't think this file should be modified as part of this PR?

Reply:
It was throwing lint errors in the build, so I just added a `|` at the end nothing else.